views/includes/trustee-macros.html

```diff
@@ -1,11 +1,11 @@
 {% macro readableTrusteeRole(role) %}
-  {% if role === 'Beneficiary' %}
+  {% if role === 'BENEFICIARY' %}
```

Comment:
we'll need some tests around the html changes so I can look at adding those myself

Reply:
added 
test/controllers/update/update.manage.trusts.review.legal.entities.controller.spec.ts
test/controllers/update/update.manage.trusts.review.individuals.controller.spec.ts